translations/en-us.yml

```diff
@@ -1356,6 +1356,20 @@ registries:
                 osfstorage: 'OSF Storage'
             empty_folder: 'This folder is empty.'
             breadcrumbs: 'Breadcrumb'
+            help_modal:
+                heading: 'Using the OSF Files Browser'
+                download_all: 'Download All Files As a Zip'
+                download_all_detail: 'Navigate to the “Files” tab in the registration navigation on the left, and then click “Download this folder”.'
+                download_folder: 'Download a Folder as Zip'
+                download_folder_detail_first: 'To download a specific folder as a zip, click the vertical ellipses'
+                download_folder_detail_second: 'to open the dropdown. Click “Download”.<br>Alternatively, open the folder and then click “Download this folder”.'
```

Comment:
It's not worth changing, but for the future, better to use `<br />` rather than just `<br>`.